include/xrpl/protocol/detail/features.macro

```diff
@@ -97,6 +97,7 @@ XRPL_FEATURE(FlowCross,                  Supported::yes, VoteBehavior::DefaultYe
 XRPL_FEATURE(Flow,                       Supported::yes, VoteBehavior::DefaultYes)
 XRPL_FEATURE(OwnerPaysFee,               Supported::no,  VoteBehavior::DefaultNo)
 XRPL_FEATURE(AMMClawback,                Supported::yes,  VoteBehavior::DefaultYes)
+XRPL_FEATURE(Credentials,                Supported::yes, VoteBehavior::DefaultNo)
```

Comment:
These last two should be at the top of the list.

Reply:
moved